example/src/main/tut/typeclass/Debug.md

```diff
@@ -40,3 +40,14 @@ Maybe.just(Foo(1)).debug
 ```
 
 *Note that the `Debug` instance for `Maybe` resolves against the `Debug` instance for `Foo`.*
+
+# String interpolator
+
+Scalaz contains a custom string interpolator that takes advantage of `Debug`.
+Inside the `z` String interpolation, only objects that have `Debug` instances can be used.
+
+```tut
+val foo = Foo(123)
+
+val interpolated = z"${foo}"
+```
```

Comment:
possibly a block demonstrating that without the `Debug` instance, compilation fails?